main.tf

```diff
@@ -0,0 +1,73 @@
+resource "azurerm_subnet" "this" {
+  name                 = "${var.cluster_name}-snet"
+  resource_group_name  = var.resource_group_name
```

Comment:
I think yo forgot to commit `variables.tf`

Reply:
Do you mean [this file](https://github.com/camptocamp/devops-stack-module-cluster-aks/blob/feature/initial_implementation/variables.tf)?

Reply:
I know what happened! I mistakenly added the file on the first commit to the module, so it is already in the main branch and the PR has no changes because I did not modify anything on the file.